src/Cake.NuGet/Install/NuGetPackageInstaller.cs

```diff
@@ -165,5 +168,26 @@ private static NuGetVersion GetNuGetVersion(PackageReference package, NuGetSourc
             }
             return null;
         }
+
+        private static (string root, string fileName) GetNuGetConfigPath(ICakeEnvironment environment, ICakeConfiguration config, ILogger log)
+        {
+            var nugetConfigFile = config.GetValue(Constants.NuGet.ConfigFile);
+            string rootPath;
+            string configFileName;
+            if (!string.IsNullOrEmpty(nugetConfigFile))
+            {
+                rootPath = Path.GetDirectoryName(nugetConfigFile);
+                configFileName = Path.GetFileName(nugetConfigFile);
+            }
+            else
+            {
+                rootPath = GetToolPath(environment, config);
+                configFileName = null;
+            }
+
+            log.LogDebug($"Found NuGet.config at: {rootPath}{Path.DirectorySeparatorChar}{configFileName}");
```

Comment:
Move logging to `NuGetPackageInstaller` instead.

Reply:
Move it where? We are in NuGetPackageInstall.cs here. Or do you mean into the ctor of it?